homeassistant/components/dlna_dms/config_flow.py

```diff
@@ -0,0 +1,177 @@
+"""Config flow for DLNA DMS."""
+from __future__ import annotations
+
+import logging
+from pprint import pformat
+from typing import Any, cast
+from urllib.parse import urlparse
+
+from async_upnp_client.profiles.dlna import DmsDevice
+import voluptuous as vol
+
+from homeassistant import config_entries
+from homeassistant.components import ssdp
+from homeassistant.const import CONF_DEVICE_ID, CONF_HOST, CONF_URL
+from homeassistant.data_entry_flow import AbortFlow, FlowResult
+from homeassistant.exceptions import IntegrationError
+
+from .const import DEFAULT_NAME, DOMAIN
+
+LOGGER = logging.getLogger(__name__)
+
+
+class ConnectError(IntegrationError):
```

Comment:
We should not inherit from `IntegrationError` here. That exception is used for a special group of exceptions by our helpers. Please inherit `HomeAssistantError` instead.

Reply:
`ConnectError` doesn't seem used.